libs/dh/market-participant/feature-edit-organization/src/lib/market-participant-edit-organization/master-data/dh-market-participant-organization-master-data.component.html

```diff
@@ -113,4 +113,17 @@
       </watt-error>
     </watt-form-field>
   </div>
+
+  <div class="form-container">
+    <watt-form-field class="field-comment">
+      <watt-label>{{ t("commentLabel") }}</watt-label>
+      <textarea
+        wattInput
+        class="textarea-comment"
+        [(ngModel)]="changes.comment"
+        (ngModelChange)="onModelChanged()"
+        #commentInputModel="ngModel"
+      ></textarea>
+    </watt-form-field>
+  </div>
```

Comment:
question: the `feature-edit-organization` library doesn't contain any tests. The component must be refactored at some point as well because it uses a pattern that we don't encourage (i.e. the `emit` in `ngOnChanges` part). Adding overall tests and general refactoring feels a bit out of scope for the PR, so can you at least create an issue on your backlog to track that?

Reply:
I'll create a maintenance issue in our backlog on this. @dzhavat